drivers/wifi/nrfwifi/Kconfig.nrfwifi

```diff
@@ -130,15 +130,12 @@ config NRF_WIFI_IF_AUTO_START
 
 config NRF_WIFI_PATCHES_BUILTIN
 	bool "Store nRF70 FW patches as part of the driver"
-	default y
+	default y if !NRF_WIFI_BUILD_ONLY_MODE
```

Comment:
I think you can say here
`default y`
as the "depends on ..." will prevent the option to be enabled.